src/qibocal/protocols/characterization/flux_dependence/resonator_flux_dependence.py

```diff
@@ -24,10 +25,72 @@ class ResonatorFluxParameters(Parameters):
     """Width for frequency sweep relative to the readout frequency [Hz]."""
     freq_step: int
     """Frequency step for sweep [Hz]."""
-    bias_width: float
+    bias_width: Optional[float] = None
     """Width for bias sweep [V]."""
-    bias_step: float
+    bias_step: Optional[float] = None
     """Bias step for sweep [a.u.]."""
+    flux_amplitude_start: Optional[Union[float, List[float]]] = None
+    """Amplitude start value(s) for flux pulses sweep relative to the qubit sweetspot [a.u.]."""
```

Comment:
Should we describe how will it behave when you pass a list or a float with several qubits ?

```suggestion
    flux_amplitude_start: Optional[Union[float, List[float]]] = None
    """Amplitude start value(s) for flux pulses sweep relative to the qubit sweetspot [a.u.]."""
```